draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -171,6 +163,47 @@
     </section>
   </section>
 
+      <section title="Terminology" anchor="terminology">
+         <t>
+           The following terms are used with special meanings.
+         </t>
+         <t>
+           <list style="hanging">
+                <t hangText="Provider:">
+                   The term "provider" is synonymously with "transit provider", see <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>.
+                </t>
+
+                <t hangText="Customer:">
+                   An autonomous system uses a provider for transit, see <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>.
+                </t>
+
+                <t hangText="CAS:">
+                  A Customer AS, see <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>.
+                </t>
+
+                <t hangText="PAS:">
+                  A Provider AS, see  <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>.
+                </t>
+
+                <t hangText="Route is ineligible:">
+                   The term has the same meaning as in <xref target="RFC4271"/>, i.e., "route is ineligible to be installed in Loc-RIB and will be excluded from the next phase of route selection."
+                </t>
+
+                <t hangText="SPAS:">
+		   A Set of Provider ASes, see <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="3"/>.
+                </t>
+
+                <t hangText="VAP:">
+		   A validated ASPA payload, see <xref target="ASPA"/>.
+                </t>
+
+                <t hangText="VAP-SPAS:">
+		   A verified ASPA payload of a set of provider ASes, see <xref target="ASPA"/>.
```

Comment:
s/verified ASPA payload/Validated ASPA Payload/ ?

Reply:
agreed and fixed